app/src/api/services/leaderboard.ts

```diff
@@ -0,0 +1,39 @@
+import db from '../../db';
+
+async function getLeaders(number_limit: number, time_limit: number) {
```

Comment:
as far as I understand, this query will be run on the logs table every time the leaderboard page is displayed. Just wondering about the impact on the server's performance? Seeing that the logs table is / will become potentially quite huge with time...

Reply:
I tested the implementation in my dev environment, against a database dump from the production server (dumped circa September) without issues, despite the limited resources of my virtual dev environment. 

So I would suggest merging it and seeing how it performs.